docs/doxygen/images/http_demo_s3_download_multithreaded.pu

```diff
@@ -0,0 +1,80 @@
+@startuml
+skinparam classFontSize 8

```

Comment:
One last nitpick can you add `skinparam dpi 300` to this image so it's not blurry on large monitors